### DISCUSSION TITLE
分布式部署的文档哪里还能看到啊？

### DISCUSSION THREAD

**Discussion by github-yxb** [kbengine/kbengine#1483] [Q&A]
谷歌百度找了几遍 论坛也打不开了 哪里还能看到分布式部署的操作?

**Comment by ybyaoyao** [comment]
论坛里KBE发过一个以前老论坛的文字整合包，你找找看吧。没图片但是有以前的文本。

**Reply by github-yxb** [reply]
thanks 
通过网上的资料、报错信息、引擎源码 已经可以了

**Reply by smilingmoon** [reply]
在哪能找到这些资料呢，可以分享下吗
